tests/test_tracking_graph.py

```diff
@@ -254,26 +204,44 @@ def test_get_and_set_node_attributes(simple_graph):
         "is_tp_division": True,
     }
 
-    simple_graph.set_node_attribute("1_0", NodeAttr.FALSE_POS, value=False)
+    simple_graph.set_flag_on_node("1_0", NodeFlag.FALSE_POS, value=False)
     assert simple_graph.nodes()["1_0"] == {
         "id": "1_0",
         "t": 0,
         "y": 1,
         "x": 1,
-        NodeAttr.FALSE_POS: False,
+        NodeFlag.FALSE_POS: False,
     }
+
+    simple_graph.set_flag_on_all_nodes(NodeFlag.FALSE_POS, value=False)
+    for node in simple_graph.nodes:
+        assert simple_graph.nodes[node][NodeFlag.FALSE_POS] is False
+
+    simple_graph.set_flag_on_all_nodes(NodeFlag.FALSE_POS, value=True)
+    for node in simple_graph.nodes:
+        assert simple_graph.nodes[node][NodeFlag.FALSE_POS] is True
```

Comment:
would be good to check also the `node_by_flag` dictionary here, and same below for edges, to ensure it's updated correctly